packages/ui-core/src/components/Button/Button.tsx

```diff
@@ -1,19 +1,28 @@
 import * as React from 'react';
-import * as PropTypes from 'prop-types';
 import cnCreate from 'utils/cnCreate';
 import './Button.less';
 import Spinner from 'docIcons/spinner.svg';
+import Arrow from 'icons/System/32/Arrow_right_32.svg';
 import detectTouch from 'utils/detectTouch';
 
 export interface IButtonProps {
-    /** Special view */
-    customView?: 'two-lines';
+    /** Custom classes for button and button text-area */
```

Comment:
```suggestion
    /** Custom classes for button and button content */
```
Не уверен, что text-area это содержимое контейнера кнопки.